process faults {2: ({6, 2}, 1666330945)}
network faults {5: [{0, 5}, {1}, {4, 2}, {3, 6}]}
{10: ("0631D15C182F9AA731645640E67AE60C264495C8C6400518B912193EFDA09D4A", {3, 5, 1, 0, 4, 6, 2}), 6: ("59CEB6BB0F77B63F5B0ECED95C40BD71FF4985D0CE52F6E560791C761CB8D6B3", {6, 3, 0, 5, 4, 2, 1}), 8: ("5C188371F2946AC5EB7E3A80E49C312496F446958F48544E4F99F8648AD9596A", {4, 0, 1, 2, 3, 6, 5}), 5: ("A8DC81AA3B8254EFCE04806F7C23701D3B12969E2C3CC3B57E73C593C0CD01A2", {1, 0, 6, 4, 3, 2, 5}), 7: ("85FD9AD1B03FCF8EA9F03D67A766599909FACCD9CBDA2EDFB5505215631A3CAA", {3, 2, 6, 0, 4, 1, 5}), 9: ("3730212DB53F26FD0AD7D25701BFA678637E6BCA841067A5688ADB054AA5D3B0", {1, 3, 4, 5, 6, 2, 0})}
7
reason: all committed
done!
